define-test.js

```diff
@@ -1368,3 +1369,41 @@ QUnit.test('define() should add a CID (#246)', function() {
 	var g = new Greeting();
 	QUnit.ok(g._cid, "should have a CID property");
 });
+
+QUnit.test("warn on using a Constructor for small-t type definintions", function() {
```

Comment:
Need to skip this in production like the other one.

Reply:
Good catch.  I'll fix that.